pages/home.html

```diff
@@ -62,15 +65,16 @@
     </a>
     Features
   </h2>
-  <ul>
-    <li>Simulates a 1-on-1 conversation with a tutor.</li>
-    <li>Able to give targeted feedback to learners based on their answers.</li>
-    <li>Allows learners to leave feedback on explorations.</li>
-    <li>Explorations are embeddable in any webpage.</li>
-    <li>Explorations can be played on mobile devices (except for some of the more complex interactions).</li>
-    <li>The exploration editor supports full version history, multiple authors, parameters and analytics dashboards.</li>
-    <li>There are extension frameworks for building new interactions and classification rules.</li>
-  </ul>
+  <p>With Oppia, you can:
+    <ul>
+      <li>simulate a 1-on-1 conversation with a tutor,</li>
+      <li>give targeted feedback to learners based on their answers (as a tutor),</li>
+      <li>leave feedback on explorations (as a learner),</li>
```

Comment:
maybe we can just let it be simple, "leave feedback on explorations", I think it's clear enough. The less we write the more readers can remember :smile: